opts/env_test.go

```diff
@@ -18,13 +18,15 @@ func TestValidateEnv(t *testing.T) {
 		"env3=abc!qwe":        "env3=abc!qwe",
 		"env_4=value 4":       "env_4=value 4",
 		"PATH":                fmt.Sprintf("PATH=%v", os.Getenv("PATH")),
+		"=a":                  fmt.Sprintf("invalid environment variable: %s", "=a"),
```

Comment:
It is really weird to let `valids` map contain invalid values.
Let's use structs.